.travis.yml

```diff
@@ -21,15 +21,38 @@ jobs:
         - |
           ./scripts/get-updated-distros.sh |
             while read -r filename; do
-              if [ "$filename" == "_topic_maps/_topic_map.yml" ]; then python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.15 --no-upstream-fetch || travis_terminate 1
-
-              elif [ "$filename" == "_topic_maps/_topic_map_osd.yml" ]; then python3 build.py --distro openshift-dedicated --product "OpenShift Dedicated" --version 4 --no-upstream-fetch || travis_terminate 1
-
-              elif [ "$filename" == "_topic_maps/_topic_map_ms.yml" ]; then python3 build.py --distro microshift --product "Microshift" --version 4 --no-upstream-fetch || travis_terminate 1
-
-              elif [ "$filename" == "_topic_maps/_topic_map_rosa.yml" ]; then python3 build.py --distro openshift-rosa --product "Red Hat OpenShift Service on AWS" --version 4 --no-upstream-fetch || travis_terminate 1
-
-              elif [ "$filename" == "_distro_map.yml" ]; then python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.15 --no-upstream-fetch || travis_terminate 1
-              fi
+              case "$filename" in
+                "_topic_maps/_topic_map.yml")
+                  distro="openshift-enterprise"
+                  product="OpenShift Container Platform"
+                  version="4.15"
+                  ;;
+                "_topic_maps/_topic_map_osd.yml")
+                  distro="openshift-dedicated"
+                  product="OpenShift Dedicated"
+                  version="4"
+                  ;;
+                "_topic_maps/_topic_map_ms.yml")
+                  distro="microshift"
+                  product="Microshift"
+                  version="4"
+                  ;;
+                "_topic_maps/_topic_map_rosa.yml")
+                  distro="openshift-rosa"
+                  product="Red Hat OpenShift Service on AWS"
+                  version="4"
+                  ;;
+                "_distro_map.yml")
+                  distro="openshift-enterprise"
+                  product="OpenShift Container Platform"
+                  version="4.15"
+                  ;;
+                *)
+                  echo "Output from get-updated-distros.sh: $filename"
+                  echo -e "\033[1;33m No topic maps or distro_map modified!\033[0m"
+                  ;;
+              esac
+              python3 build_for_portal.py --distro "$distro" --product "$product" --version "$version" --no-upstream-fetch
```

Comment:
How does build_for_portal.py compare to previous version in terms of time to complete builds?

Reply:
It takes around the same time. I did some test runs, but they've been swept away by the mighty tide of Travis builds, lost in the vast ocean of continuous integration! 🌊 

I'll test again.